app/frontend/modules/search.js

```diff
@@ -110,7 +110,7 @@ export const sort = (query, options) => {
 export const suggestion = (value, options) => {
   const option = options.find((o) => o.name === value)
   if (option) {
-    const html = option.append ? `<span class="autocomplete__option__append">${value}</span> <span>${option.append}</span>` : `<span>${value}</span>`
+    const html = option.append ? `<span class="autocomplete__option__append">${option.text}</span> <span>${option.append}</span>` : `<span>${option.text}</span>`
```

Comment:
this line is appending options that meet the search to the list of suggestions in the dropdown?
I'm interested in what the value of `option.append` is anyway since I don't know how the existing code works.
What is value? I guess options an array of things that match the search in the format of the kinds of thing returned by the enhanceOption method below?
is append a hint text?

Reply:
We get append values from `answer_option_append` it's currently only relevant for schemes, it just displays how many completed/incomplete locations the scheme has. Purely for displaying the info, I don't think it's functional, different from hint text